cmd/clusterctl/client/config.go

```diff
@@ -42,8 +43,9 @@ func (c *clusterctlClient) GetProvidersConfig() ([]Provider, error) {
 	return rr, nil
 }
 
-func (c *clusterctlClient) GetProviderComponents(provider string, providerType clusterctlv1.ProviderType, targetNameSpace, watchingNamespace string) (Components, error) {
-	components, err := c.getComponentsByName(provider, providerType, targetNameSpace, watchingNamespace)
+func (c *clusterctlClient) GetProviderComponents(provider string, providerType clusterctlv1.ProviderType, options repository.ComponentsOptions) (Components, error) {
+	options.SkipVariables = true
```

Comment:
Given that we are exponing options, should we leave to choose if to set this flag or not to the CLI/Library consumers?

Reply:
Maybe this can be in a follow up PR